### ISSUE TITLE
[BUG]: requirements.txt in log-parsing-models for training and tuning models folder

### ISSUE THREAD

**Issue by nyck33** [nv-morpheus/Morpheus#1227]
### Version

23.11

### Which installation method(s) does this occur on?

Docker

### Describe the bug.

When I cd to 
```coder ➜ /workspaces/morpheus/models/training-tuning-scripts $ cd log-parsing-models
coder ➜ .../morpheus/models/training-tuning-scripts/log-parsing-models $ ls
log-parsing-training.ipynb  requirements.txt  resources
coder ➜ .../morpheus/models/training-tuning-scripts/log-parsing-models $ pip install -r requirements.txt
Defaulting to user installation because normal site-packages is not writeable
ERROR: Invalid requirement: 'onnx=1.14.0' (from line 1 of requirements.txt)
Hint: = is not a valid operator. Did you mean == ?
coder ➜ .../morpheus/models/training-tuning-scripts/lo
```
I open requirements.txt and see
```
onnx=1.14.0
seqeval==1.2.2
transformers==4.22.2

```

### Minimum reproducible example

```shell
Please see above.
```

### Relevant log output

Just a minor typo

### Full env printout

```
(morpheus) coder ➜ /workspaces/morpheus/external $ ls
morpheus-visualizations  utilities
(morpheus) coder ➜ /workspaces/morpheus/external $ cd utilities
(morpheus) coder ➜ /workspaces/morpheus/external/utilities $ ls
(morpheus) coder ➜ /workspaces/morpheus/external/utilities $ ls
(morpheus) coder ➜ /workspaces/morpheus/external/utilities $ 
```
Not sure why it's not there, in the dev container. 

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

**Comment by jarmak-nv** [comment]
Hi @nyck33!
  
  Thanks for submitting this issue - our team has been notified and we'll get back to you as soon as we can!
  In the mean time, feel free to add any relevant information to this issue.

**Comment by nyck33** [comment]
Actually I made a conda env with the requirements.txt in the log-parsing-models folder but it's first cell requires torch which is not on that file
```
from seqeval.metrics import classification_report,accuracy_score,f1_score
from transformers import BertForTokenClassification
from tqdm import tqdm,trange
from collections import defaultdict
import string
import pandas as pd
import numpy as np
import cupy
import cudf
from cudf.core.subword_tokenizer import SubwordTokenizer
import torch
import torch.nn as nn
import torch.nn.functional as F
from torch.optim import Adam
from torch.utils.data import TensorDataset, DataLoader
from torch.utils.data.dataset import random_split
from torch.utils.dlpack import from_dlpack
```
That's extremely confusing when you put a requirements.txt in a folder with a notebook, looks like almost every folder has a requirements.txt or .yml which makes it seem like "build your env using this to run this notebook" right?  

So can you pleae quickly fix this?  I'm upgrading this to critical.
